src/redux/books/books.js

```diff
@@ -0,0 +1,35 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
```

Comment:
- [ ] When creating actions, they should follow the duck pattern for creating them. For example:
`const ADD_BOOK = 'bookstore/books/ADD_BOOK'`. Kindly implement this change. 👍